velox/exec/Operator.h

```diff
@@ -194,6 +195,18 @@ class Operator {
         int32_t id,
         const std::shared_ptr<const core::PlanNode>& node) = 0;
 
+    // Translates plan node to join bridge. Returns nullptr if the plan node
+    // cannot be handled by this factory.
+    virtual std::shared_ptr<JoinBridge> translate(
```

Comment:
It is likely that we'll need to add more parameters to this method in the future, in which case the compiler may not be able to distinguish this method from the one above just by signature. Perhaps, we could use more descriptive names, e.g. toOperator, toJoinBridge and toOperatorSupplier.

Reply:
Renamed